CliFx/Models/Extensions.cs

```diff
@@ -90,7 +91,7 @@ public static IReadOnlyList<string> GetAliases(this CommandOptionSchema optionSc
         /// <summary>
         /// Gets whether a command was specified in the input.
         /// </summary>
-        public static bool IsCommandSpecified(this CommandInput commandInput) => !string.IsNullOrWhiteSpace(commandInput.CommandName);
+        public static bool IsCommandSpecified(this CommandInput commandInput) => commandInput.Arguments.Any();
```

Comment:
I guess this is not accurate anymore because arguments being there doesn't guarantee that the command was specified. I think this should be removed and the callsites need to be adjusted to account for arguments now.